utils/apiClient.ts

```
@@ -0,0 +1,59 @@
+// Encrypt request payload
+apiClient.interceptors.request.use(async (config) => {
+  // If skipEncryption is true, don't encrypt
+  if ((config as any).skipEncryption) {
+    return config;
+  }
+
+  if (config.data instanceof FormData) {
+    try {
+      // Convert FormData to an object first
+      const formDataObj: Record<string, any> = {};
+      for (let [key, value] of config.data.entries()) {
```

ESLint

[error] 'key' is never reassigned. Use 'const' instead.

(prefer-const)